test/nuts/blank.nut.ts

```diff
@@ -0,0 +1,6 @@
+/*
+ * Copyright (c) 2021, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
```

Comment:
is there going to be a NUT in a future WI?

Reply:
If not, you can comment it out of the config.yml

Reply:
I can see value in having a NUT that would run `sfdx update` and make sure the post install hooks call the `info:releasenotes:display` command. But it would make more sense for those NUTs to live in cli repos or in the [install NUTs](https://github.com/salesforcecli/plugin-release-management/blob/main/src/commands/cli/install/test.ts). Can probably just remove the circle config. Can you think of any potential NUTs that could run in this repo?

Reply:
I'd run the command from a NUT on a few of the known helpers (but only if the above changes of throwing from the command happen).

We do run `sfdx update` as part of the build verification scripts--but it probably wouldn't make a difference since the command is written in a way that it never has errors.